packages/database/src/migrations/20210205033535-AddSTRIVEMolecularDataOverlays-modifies-data.js

```diff
@@ -41,113 +41,70 @@ const BASE_OVERLAY = {
   projectCodes: '{strive,explore}',
 };
 
-const STR_QMAL05_OVERLAY = {
-  ...BASE_OVERLAY,
-  id: generateId(),
-  name: 'QMAL Positive',
-  measureBuilderConfig: {
-    dataValues: {
-      STR_QMAL05: 1,
-    },
+const OVERLAYS_TO_IMPORT = [
+  {
+    id: 'STR_QMAL05_POSITIVE',
```

Comment:
This is very nitpicky, but the existing overlay in that group has id: `PG_STRIVE_K13_C580Y_Positive`. Did you have a particular reason to deviate from that pattern?